src/App.jsx

```diff
@@ -2,6 +2,7 @@
 import { PhoneIcon } from '@chakra-ui/icons';
 import React from 'react';
 import './App.css';
+import DonationForm from './components/DonationForm';
```

Comment:
🚫 **[eslint]** <[import/no-named-as-default-member](https://github.com/import-js/eslint-plugin-import/blob/v2.26.0/docs/rules/no-named-as-default-member.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parse errors in imported module './components/DonationForm': Unexpected token . (39:44)